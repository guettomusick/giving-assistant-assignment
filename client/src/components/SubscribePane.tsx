import React from 'react';

const SubscribePane = () => {
  return (
    <div>SubscribePane</div>
  )
};

export default SubscribePane;