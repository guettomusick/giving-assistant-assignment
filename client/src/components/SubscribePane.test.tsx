import React from 'react';
import renderer from 'react-test-renderer';

import SubscribePane from './SubscribePane';

describe('SubscribePane', () => {
  it('renders', () => {
    const component = renderer.create(
      <SubscribePane/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
