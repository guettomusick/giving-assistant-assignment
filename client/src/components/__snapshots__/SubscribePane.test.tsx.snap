// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscribePane renders 1`] = `
<div
  className="sc-AxgMl iVgSIG"
>
  <div
    className="sc-AxheI fRuFYU"
  >
    <div
      className="sc-Axmtr gxALLi"
    />
    <div
      className="sc-Axmtr gxALLi"
    />
  </div>
</div>
`;
